### PR TITLE
webui: Also filter languages by English names

### PR DESCRIPTION
**Pull request by marusak**

### CODE REVIEW

File: ui/webui/test/check-language [hunk 1]
```diff
@@ -57,10 +57,17 @@ class TestLanguage(MachineCase):
         b.wait_visible("#English--English-")
         b.wait_visible("#Deutsch--German-")
 
-        # Check that the language menu shows filtered menu entries
+        # Check filtering on English names
+        b.set_input_text(f"#{i.welcome_id}-menu-toggle-select-typeahead", "cze")
+        b.wait_not_present("#Deutsch--German-")
+        b.wait_not_present("#English--English-")
+        b.wait_visible("#installation-language-option-cs_CZ")
```
Comment (KKoukiou) [inline ui/webui/test/check-language:64]: Please use (f"#{i.welcome_id}-option-cs_CZ")

File: ui/webui/src/components/localization/InstallationLanguage.jsx [hunk 2]
```diff
@@ -88,6 +89,39 @@ class LanguageSelector extends React.Component {
...
     }
 
+    renderOptions (filter) {
+        const { languages, locales } = this.state;
+        const idPrefix = this.props.idPrefix;
+        const filterLow = filter.toLowerCase();
+
+        return locales.reduce((filtered, langLocales) => {
+            const currentLang = languages.find(lang => getLanguageId(lang) === getLanguageId(langLocales[0]));
+            console.log("Processing", langLocales[0], currentLang);
```
Comment (KKoukiou) [inline ui/webui/src/components/localization/InstallationLanguage.jsx:99]: Is this leftover from debugging?

File: ui/webui/test/check-language (hunk 1 above)
Comment (marusak) [reply ui/webui/test/check-language]: Updated

File: ui/webui/test/check-language [hunk 3]
```diff
@@ -57,10 +57,17 @@ class TestLanguage(MachineCase):
...
+        b.set_input_text(f"#{i.welcome_id}-menu-toggle-select-typeahead", "cze")
+        b.wait_not_present("#Deutsch--German-")
+        b.wait_not_present("#English--English-")
+        b.wait_visible(f"#{i.welcome_id}-option-cs_CZ")
+
+        # Check filtering on native names
         b.set_input_text(f"#{i.welcome_id}-menu-toggle-select-typeahead", "Deutsch")
-        b.wait_visible("#Deutsch--German-")
         b.wait_not_present("#English--English-")
+        b.wait_not_present("#installation-language-option-cs_CZ")
```
Comment (KKoukiou) [inline ui/webui/test/check-language:69]: Sorry, can you please parameterize this as well as you did for above?

Comment (marusak) [reply ui/webui/test/check-language:69]: argh sorry :/ of course!

Comment (marusak) [reply ui/webui/test/check-language]: done

**Review by KKoukiou — APPROVED** [review]

**Comment by marusak** [comment]
@M4rtinK @VladimirSlavik Is this blocked on anything?

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Comment by KKoukiou** [comment]
Merging without 3rd review - as it's a bug fix and nice to have in for the demo.

**Pull request merged**